XcodeServerSDK.xcodeproj/project.pbxproj

```diff
@@ -197,6 +239,7 @@
 			children = (
 				3A7B48BC1B2A5AC40077ABEA /* XcodeServerSDK */,
 				3A7B48C91B2A5AC40077ABEA /* XcodeServerSDKTests */,
+				11376BD11B3A2F910005A681 /* XcodeServerSDK - ᴡᴀᴛᴄʜ */,
```

Comment:
This folder holds the public header for the `ᴡᴀᴛᴄʜ` framework (its different than the other ones; unless we provide target conditionals